test/ejercicio1b.test.js

```diff
@@ -0,0 +1,156 @@
+const ejercicio1b = require('../src/ejercicio1b');
+//Tests of evenCounting 
```

Comment:
How about putting each one of the tests in describe blocks ?

what do you think